modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/ReactFDSDisplayContext.java

```diff
@@ -15,7 +15,7 @@
 package com.liferay.frontend.data.set.sample.web.internal.display.context;
 
 import com.liferay.frontend.data.set.sample.web.internal.constants.FDSSampleFDSNames;
-import com.liferay.frontend.data.set.sample.web.internal.servlet.ServletContextUtil;
+import com.liferay.frontend.data.set.sample.web.internal.fdsviewserializer.util.FDSViewSerializerUtil;
```

Comment:
@W2jx Can you put this class in `com.liferay.frontend.data.set.sample.web.internal.view.util`? The interface and impl are all in `com.liferay.frontend.data.set.*.view`.